packages/webapp/pages/devcard.tsx

```diff
@@ -130,13 +130,13 @@ const Step2 = ({
     <>
       <div className="flex flex-col self-stretch laptop:self-center items-center mx-2">
         <Tilt
-          className="self-stretch laptop:w-96"
+          className="self-stretch laptop:w-96 overflow-hidden"
           glareEnable
           perspective={1000}
           glareMaxOpacity={0.25}
           glarePosition="all"
           trackOnWindow
-          style={{ transformStyle: 'preserve-3d' }}
+          style={{ transformStyle: 'preserve-3d', borderRadius: '35px' }}
```

Comment:
According to our designer, the border radius should be `26px`. Also if possible please use `rm` units
Please fix it and let's merge :)

Reply:
No problem! Looking into this a bit more, 26px definitely is not enough. I added a red background for contracts. The first image is 26px

![Screenshot 2021-08-24 at 09 54 31](https://user-images.githubusercontent.com/11774156/130588195-09299c0f-d08b-4627-8d02-0b601ecd2e8b.png)

This image is 35px which is the current commit

![Screenshot 2021-08-24 at 09 54 41](https://user-images.githubusercontent.com/11774156/130588269-c2b98a75-1973-4efc-8117-0b087db7503e.png)

And to keep things tidy, the updated commit has a 2rem (32px) border-radius. Shown below with and without the red background.

![Screenshot 2021-08-24 at 09 54 49](https://user-images.githubusercontent.com/11774156/130588445-acf184b4-b6c9-4a73-b74a-b75949087735.png)
![Screenshot 2021-08-24 at 09 54 41](https://user-images.githubusercontent.com/11774156/130588448-b7560e9b-e83f-4dca-85d4-6c0ab6e326d0.png)


is that ok or should I amend further?

Reply:
Oh, that might happen because of the 3d effect. Let's go with 2rem. Thanks a lot for validating :)

Reply:
no problem!